src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurStandardAccountingExtractCollectorBatchBuilder.java

```diff
@@ -257,6 +257,11 @@ protected boolean shouldProcessLine(ConcurStandardAccountingExtractDetailLine sa
                 return true;
             case ConcurConstants.PAYMENT_CODE_UNIVERSITY_BILLED_OR_PAID :
                 reportCorporateCardPayment(saeLine);
+                // Temporary hack to ignore personal expense lines until their handling has been implemented.

```

Comment:
please remove the comment. Is the handling of personal expense lines going to be implemented as part of KFSPTS-8359.